Corona-Warn-App/src/test/java/de/rki/coronawarnapp/submission/ExposureKeyHistoryCalculationsTest.kt

```diff
@@ -83,18 +69,51 @@ class ExposureKeyHistoryCalculationsTest {
 
     @Test
     fun test_toExternalFormat() {
```

Comment:
Ask: I think this needs more test cases. E.g. a test case with a TEK missing in the sequence, test cases with multiple TEKs on the same day, etc. Might be worth taking a look at the iOS test cases, e.g. starting https://github.com/corona-warn-app/cwa-app-ios/blob/d539979edea684be45dda527d52b9e846d618671/src/xcode/ENA/ENA/Source/Extensions/__tests__/ENTemporaryExposureKey%2BProcessingTests.swift#L211